civcraftConfigs/gimmelconfig.yml

```diff
@@ -366,3 +367,37 @@ monster:
       - DIRT
       - GRASS
       - SAND
+ AquilaSection:
+  updatetime: 2m
+  areas:
+   locations:
+    AquilaRuins:
+     shape: CIRCLE
+     xsize: 100
+     center:
+      x: 1000
+      z: 64
+  mobconfig:
+   GhostWitch:
+    identifier: FloatingWitch2
+    spawn: WITCH
```

Comment:
I believe it works based on how I read the documentation https://www.reddit.com/r/Civcraft/comments/4fql31/an_idea_for_mob_spawning/d2bgka0 

Notice that this section you are looking at is under monster: and not under spawner: This time. I hope that it makes some witches spawn.